tests/test_schema.py

```diff
@@ -290,3 +298,34 @@ def test_schema_validate_json_schema_with_bot_uri_override(tmp_path):
             os.environ["CONDA_SMITHY_BOT_SCHEMA_URI"] = old_val
         else:
             del os.environ["CONDA_SMITHY_BOT_SCHEMA_URI"]
+
+
+@pytest.mark.xfail(
+    reason=(
+        "rattler-build-conda-compat makes global modifications to ruamel.yaml"
+        " - see https://github.com/prefix-dev/rattler-build-conda-compat/issues/88"
+    ),
+)
+def test_schema_with_rattler_build_conda_compat():
+    from rattler_build_conda_compat.yaml import _yaml_object
+
+    _yaml_object()
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        pth = os.path.join(tmpdir, "conda-forge.yml")
+        with open(pth, "w") as fp:
+            fp.write(
+                textwrap.dedent(
+                    """\
+                bot:
+                version_updates:
+                    random_fracvtion_to_keep: 0.02
```

Comment:
Or is the typo ("fracVtion") here intentional?

Reply:
This is a legit typo. lol.

Reply:
The linter is two parts, one for the conda-forge.yaml and one for the recipe. The offending code in rattler build is imported when we lint the recipe. So for this test, I put the offending code in rattler build import at the top. 

In real life feedstocks, that import comes after the conda-forge.yaml linter because I swapped the order of the linting steps. 


Global state sucks which is why this is confusing.

Reply:
OK, that makes sense, thanks! :)